### ISSUE TITLE
Kits: Default tool selection

### ISSUE THREAD

**Issue by textshell** [jeandet/qt-creator#4]
Currently auto detected tools are preferred to custom tools. That seems backwards. Maybe introduce a "isDefault" for the tools.

**Comment by jeandet** [comment]
Yep makes sense, I propose to add a "Make Default" button like CMake plugin does.


![Screenshot from 2020-05-01 14-47-20](https://user-images.githubusercontent.com/7570249/80806397-e0ce0f00-8bba-11ea-8c26-2b4463781877.png)
